pig/src/main/kotlin/org/partiql/pig/domain/Utils.kt

```diff
@@ -89,15 +89,15 @@ internal fun DataType.toIonElement(includeTypeTag: Boolean): IonElement = when (
                 ionSymbol(tag),
                 *namedElements.map { it.toIonElement(this.tupleType) }.toTypedArray()
             )
-        )
+        ).withAnnotations(annotations.map { it.toString().toLowerCase() })
```

Comment:
Why do we need those annotation in lower case? It seems like we deliberately convert them to upper case by calling the `of` function. 
The root question is, why would the case sensitivity matter in this case?

Reply:
These methods are used to output the type domain into Ion again. All identifiers are lower snake case. Since the annotations are enums, I don't have to worry about snake case, only lowercasing